source/plugins/go/src/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	github.com/Azure/azure-kusto-go v0.5.2
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
+	github.com/Microsoft/go-winio v0.5.2
```

Comment:
version - 0.6.0 available. and also lets ping the owners (MS folks) of the repo to ensure this can be used for the production or is there any pending to use?

Reply:
Yes I will ping the folks and update the version in our final merge with all the testing